File: servicio-orquestador/src/main/java/co/edu/unicauca/orquestador/fachadaServices/DTORespuesta/RespuestaConsultaPazySalvoLaboratorioDTO.java
package co.edu.unicauca.orquestador.fachadaServices.DTORespuesta;

import java.time.LocalDate;

import lombok.Data;

@Data
public class RespuestaConsultaPazySalvoLaboratorioDTO {
    private int codigoEstudiante; 
    private LocalDate fechaPrestamo;
    private String estado; 
    private LocalDate fechaDevolucionEstimada; 
    private LocalDate fechaDevolucionReal; 
    private String equipo; 
}

File: app-estudiante/src/app/modelo/RespuestaConsultaPazySalvoLaboratorioDTO.ts
export class RespuestaConsultaPazySalvoLaboratorioDTO {
  codigoEstudiante!: number;
  fechaPrestamo!: string;             // formato 'YYYY-MM-DD'
  estado!: string;
  fechaDevolucionEstimada!: string;   // formato 'YYYY-MM-DD'
  fechaDevolucionReal!: string;       // formato 'YYYY-MM-DD'
  equipo!: string;
}
